### DISCUSSION TITLE
Difference between Hubs Cloud Personal or Enterprise vs Free Account?

### DISCUSSION THREAD

**Discussion by jl3471** [mozilla/hubs#2725] [General Questions & Discussion] [answered]
Hi all, I have been experimenting and making various scenes in Spoke and I'm impressed with Mozilla Hubs.  Question: What would be the reason or key benefits to switch to the paid version of Hubs Cloud?   I'm asking because funds are really limited and I wanted to see if it's worth it or not.

**Comment by misslivirose** [comment, marked as answer]
Hubs Cloud is the platform to use if you want more granular control over things at the administrator level (for example, being able to manage system-wide accounts, the branding and behavior of the platform), the ability to point your own domain name to the platform, and being able to host the entire platform on an account that you own. Hubs Cloud is essentially taking hubs.mozilla.com and allowing you to run a version of that yourself on AWS, which may or may not be necessary for the use case(s) you have in mind.

**Reply by emclaren** [reply]
I'm not sure if this is the same person who asked this in Discord 👋 but I drafted up this response for them that I thought might be helpful to include here. 

hubs.mozilla.com is our free version, Hubs Cloud is the self-hosted version. There are a few reasons why Hubs Cloud might be the direction you wish to go : 

* Customizable branding - hubs.mozilla.com has Hubs' colors & logos, with Hubs Cloud you can completely personalize this
* Custom domain - want your site to live at "my-custom-social-vr-world.com"? you can buy your own domains to use with Hubs Cloud
* Access to the code base - the Hubs code lives on your server, so you can build custom components to load on top
* Analytics - since you have access to the code you can add your own analytics tracking 
* Custom list of avatars & scenes - while the scenes and avatars are customizable on hubs.mozilla.com, users have access to our full database. You can limit or personalize the scenes on Hubs Cloud so they fit with your event (maybe you want special branding on all the avatars) 
* Custom landing page with public rooms - on Hubs Cloud, you can make rooms public so they show up on the landing page. This is nice if you are having an event spread across multiple rooms and want users to be able to find them easily. 
* Limit access - you can upload a list of approved users who can access the space to Hubs Cloud,  keeping your rooms private
* Data Privacy/ownership - as the server owner, you have control over how data is stored & for how long etc. Nothing passes through Mozilla's servers. This is very important for some organizations.

**Comment by robinkwilson** [comment]
Great question! Here's a short blurb on the difference between the free account (hubs.mozilla.com) vs hubs cloud: [Hubs Cloud FAQ](https://hubs.mozilla.com/docs/hubs-cloud-faq.html#why-use-hubs-cloud-vs-hubsmozillacom) and [on HC Personal vs HCC Enterprise](https://hubs.mozilla.com/docs/hubs-cloud-faq.html#personal-vs-enterprise)

Also to add to the above.

**Minimizing HC costs docs**
If you decide to try HC, I highly recommend you get used to turning **"Offline" mode** on and off (don't set it for your first deploy) and enable **"database pausing"**. Minimizing cost documentation + User story for minimizing costs + descriptions of those two settings is here: [Hubs Cloud minimizing costs](https://hubs.mozilla.com/docs/hubs-cloud-aws-costs.html#minimize-your-costs---a-user-story)
